src/data/community-events.json

```
@@ -179,6 +179,15 @@
     "startDate": "2022-06-09",
     "endDate": "2022-06-12"
   },
+  {
+    "title": "HODLerCon",
+    "to": "https://hodlercon.com",
+    "sponsor": null,
+    "location": "Waikiki, HI, USA",
```

For USA based events we have a convention of 'State, COUNTRY'

```suggestion
    "location": "Hawaii, USA",
```